dev/integration_tests/android_engine_test/test_driver/hcpp/platform_view_main_test.dart

```diff
@@ -63,4 +63,16 @@ void main() async {
       matchesGoldenFile('$goldenPrefix.platform_view_portait_rotated_back.png'),
     );
   }, timeout: Timeout.none);
+
+  // Note: this doesn't reset the app so if additional test cases are added
+  // make sure to press the button again.
```

Comment:
You could just hit the button as a result of finishing the test, right?

```dart
addTearDown(() async {
  // Reset the state regardless of the test passing or not.
  await flutterDriver.tap(find.byValueKey('AddOverlay'));
});
```

Reply:
done